docs/en/stack/ml/df-analytics/ml-supervised-workflow.asciidoc

```diff
@@ -0,0 +1,139 @@
+[role="xpack"]
+[[ml-supervised-workflow]]
+= Introduction to supervised learning
+
+Elastic supervised learning allows you to train a {ml} model based on training 
+examples that you provide. You can then use your model to make predictions on 
+new data. This page describes the end-to-end workflow to clarify how you can 
+train a model, evaluate it and deploy it. It gives a high-level overview of the 
+steps required to identify and implement a solution using supervised learning.
+
+The workflow for supervised learning consists of the following stages:
+
+image::images/ml-dfa-lifecycle-diagram.png["Supervised learning workflow"]
+
+These are iterative stages, meaning that after evaluating each step, you might 
+need to make adjustments before you move further.
+
+
+[[define-problem]]
+== Define the problem
+
+It’s important to take a moment and think about where {ml} can be most 
+impactful. Consider what type of data you have available and what value it 
+holds. The better you know the data, the quicker you will be able to create {ml} 
+models that generate useful insights. What kinds of patterns do you want to 
+discover in your data? What type of value do you want to predict: a category, or 
+a numerical value? The answers help you choose the type of analysis that fits 
+your use case.
+
+After you identify the problem, consider which of the {ml-features} are most 
+likely to help you solve it. Unsupervised learning – like {anomaly-detect} or 
+{oldetection} – does not require a labelled data set to train the model on while 
+supervised learning requires a data set that contains the known values that the 
+model can be trained on.
+
+{stack} provides the following types of supervised learning: 
+
+* {regression}: predicts **continuous, numerical values** like the response time 
+  of a web request. 
+* {classification}: predicts **discrete, categorical values** like whether a 
+  https://www.elastic.co/blog/machine-learning-in-cybersecurity-training-supervised-models-to-detect-dga-activity[DNS request originates from a malicious or benign domain]. 
+
+
+[[prepare-transform-data]]
+== Prepare and transform data
+
+You have defined the problem and selected an appropriate type of analysis. The 
+next step is to produce a high-quality data set in {es} with a clear 
+relationship to your training objectives. If your data is not already in {es}, 
+this is the stage where you develop your data pipeline. If you want to learn 
+more about how to ingest data into {es}, refer to the 
+{ref}/ingest.html[documentation].
```

Comment:
I like to use the target page's title if possible, so readers have a hint of where the link will take them.
```suggestion
{ref}/ingest.html[Ingest node documentation].
```